vsts/gatedBuild.ps1

```diff
@@ -18,7 +18,7 @@ docker ps -a
 #Load functions used to check what, if any, e2e tests should be run
 . .\vsts\determine_tests_to_run.ps1
 
-$runTestCmd = ".\build.ps1 -clean -build -configuration DEBUG -framework $env:FRAMEWORK"
+$runTestCmd = ".\build.ps1 -clean -build -configuration DEBUG -framework $env:FRAMEWORK -nobuildBeforeTesting"
```

Comment:
Does casing matter on these flags? "-nobuild..." vs "-noBuild..."?

Reply:
Ah, I would assume that it is. I'll fix this.